### PR TITLE
Release a new version of the gem

### PR DESCRIPTION
**Pull request by smortex**

### CODE REVIEW

File: lib/riemann/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Riemann
-  VERSION = '0.2.6'
+  VERSION = '0.3.0'
```
Comment (smortex) [inline lib/riemann/version.rb:2]: I bumped the minor version but we still are in the 0.x era. I am wondering if it makes sense to start saying that this code is ready for production use and bump to 1.0.0?

**Comment by jamtur01** [comment]
I'm okay with that.

**Comment by smortex** [comment]
> I'm okay with that.

@jamtur01 Bumping to 1.0.0 as I suggested in the in-line comment or okay with the 0.3.0 as it is currently :grin: ?

**Comment by jamtur01** [comment]
1.0.0 sounds fine. :)

**Pull request merged**